src/main/java/org/junit/runners/Parameterized.java

```diff
@@ -253,9 +258,11 @@ private RunnersFactory(Class<?> klass) {
         private List<Runner> createRunners() throws Throwable {
             Parameters parameters = getParametersMethod().getAnnotation(
                     Parameters.class);
-            return Collections.unmodifiableList(createRunnersForParameters(
-                    allParameters(), parameters.name(),
-                    getParametersRunnerFactory()));
+            ParametersRunnerFactory runnerFactory = getParametersRunnerFactory();
+            Iterable<Object> allParameters = allParameters();
+            List<Runner> runners = createRunnersForParameters(allParameters,
+                    parameters.name(), runnerFactory);
+            return Collections.unmodifiableList(runners);
```

Comment:
Please separate refactoring commits from commits that change the classes behaviour (aka feature commits). This makes it a lot easier to understand the commit, because the reader doesn't have to find out whether your change is modifying the behaviour or not.